client/webserver/locales/pl-pl.go

```diff
@@ -163,13 +163,11 @@ var PlPL = map[string]string{
 	"Restore from seed":           "Przywróć z ziarna",
 	"Import Account":              "Importuj konto",
 	"no_wallet":                   "brak portfela",
-	"create_a_x_wallet":           "Utwórz portfel {{.Name}}", // .Name will be an asset symbol like DCR
+	"create_a_x_wallet":           "Utwórz portfel <span data-asset-name=1></span>",
 	"dont_share":                  "Nie udostępniaj nikomu. Nie zgub go.",
 	"Show Me":                     "Pokaż",
 	"Wallet Settings":             "Ustawienia portfela",
 	"add_a_x_wallet":              `Dodaj portfel <img data-tmpl="assetLogo" class="asset-logo mx-1"> <span data-tmpl="assetName"></span>`,
-	"ready":                       "gotowy",
-	"off":                         "wyłączony",
```

Comment:
Surely these off and ready are still good, capitalization aside, and I think we can feel out if caps can be done on a per-language basis (pl and pt-br, but not zh?)